### PR TITLE
Also look into private method

### PR DESCRIPTION
**Pull request by maxime-carbonneau**
Add :include_private to respond_to? so the helper can be included into a controller

### CODE REVIEW

**Comment by zorab47** [comment]
The second argument to [`respond_to?`][1] is a boolean. I'm not sure if `link_to` is available within the controller. You'd likely have to call `link_to` from `view_context`, which may not achieve what you're looking for...

What's the goal of this change? 

[1]: http://ruby-doc.org/core-2.5.0/Object.html#method-i-respond_to-3F

**Comment by maxime-carbonneau** [comment]
I would like to access auto_url_for from the controller.

**Comment by maxime-carbonneau** [comment]
But I can achieve the same result by overloading the method in the controller.

**Comment by zorab47** [comment]
Looks OK to me. Thoughts @varyonic or @Fivell ?

**Comment by Fivell** [comment]
@zorab47  I'm not sure I understand the reason of this change.  Can you summarize this ?

**Comment by maxime-carbonneau** [comment]
The idea is to let the controller use the method auto_url_for :
```ruby
def destroy
  destroy! do
    redirect_to auto_url_for(resource.imageable)
  end
end
```

**Comment by javierjulio** [comment]
Not sure there's any interest or need for this. If so, we can reopen.
